### PR TITLE
Adds responsive header

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Setup a responsive header with a menu triggered at lett than of width 1024px.

### CODE REVIEW

File: src/components/Header/index.scss [hunk 1]
```diff
@@ -1,23 +1,88 @@
...
+.Header-sections {
+  display: flex;
+  flex-direction: row;
+
+  a:not(:last-child) {
+    margin-right: $size-base;
+  }
+
+  h3 {
+    margin: 0;
```
Comment (frm) [inline src/components/Header/index.scss]: This should probably be fixed by a `reset.css`

File: src/styles/variables.scss [hunk 2]
```diff
@@ -13,13 +13,34 @@ $color-red: #f95252;
...
+$fontSize-xlarge: 36px;
+$lineHeight-xsmall: 16px;
+$lineHeight-small: 20px;
+$lineHeight-medium: 28px;
+$lineHeight-large: 36px;
+$lineHeight-xlarge: 52px;
+$fontFamily: 'proxima-nova', helvetica arial, sans-serif;
+$fontWeight-light: 300;
+$fontWeight-regular: 400;
+$fontWeight-bold: 700;
```
Comment (frm) [inline src/styles/variables.scss]: The variable naming is very, very inconsistent. If the components are `Uppercase`, why are these `camelCase` and why are some even all `downcase`?  [Example](https://github.com/subvisual/2017.mirrorconf.com/blob/master/css/Theme/Global.scss)

File: src/components/Header/index.scss [hunk 3]
```diff
@@ -1,23 +1,88 @@
...
+  width: 100%;
+
+  padding: $size-base $size-xxlarge $size-base $size-xxlarge;
 
   background-color: $color-deep-blue;
 
-  h1 {
+  color: white;
+
+  h2 {
```
Comment (frm) [inline src/components/Header/index.scss]: This should go into a `reset` file

File: src/components/Header/index.js [hunk 4]
```diff
@@ -3,16 +3,81 @@ import Link from "gatsby-link";
...
+  },
+  {
+    name: "Section 3",
+    link: "#section3",
+  },
+  {
+    name: "Section 4",
+    link: "#section4",
+  },
+];
```
Comment (frm) [inline src/components/Header/index.js]: How do you feel about putting this in a `data.js` file and importing it from there?

**Review by frm — CHANGES_REQUESTED** [review]
I haven't checked how this goes into suit as well but I'd really add a `reset` file before going on. You keep using tag specific rules and these should be avoided.

File: src/components/Header/index.scss (hunk 1 above)
Comment (jfranciscosousa) [reply src/components/Header/index.scss]: :+1:

File: src/styles/variables.scss (hunk 2 above)
Comment (jfranciscosousa) [reply src/styles/variables.scss]: Will fix this later :+1:

File: src/components/Header/index.scss (hunk 3 above)
Comment (jfranciscosousa) [reply src/components/Header/index.scss]: :+1:

File: src/components/Header/index.js (hunk 4 above)
Comment (jfranciscosousa) [reply src/components/Header/index.js]: This is not going to get reused atm, so I think it's not needed right now. Better to keep things tidy.

File: src/styles/variables.scss (hunk 2 above)
Comment (jfranciscosousa) [reply src/styles/variables.scss]: Fixed.

File: src/styles/variables.scss [hunk 5]
```diff
@@ -13,13 +8,35 @@ $color-red: #f95252;
...
+$size-xxlarge: 84px;
 
 $max-tablet-size: 768px;
 
-$grid-size: 992px;
\ No newline at end of file
+$grid-size: 992px;
+
+$Logo-font-size-small: 12px;
+$Logo-padding: 2px;
```
Comment (frm) [inline src/styles/variables.scss]: These should go into the logo component.

**Review by frm — CHANGES_REQUESTED** [review]

File: src/styles/variables.scss (hunk 5 above)
Comment (jfranciscosousa) [reply src/styles/variables.scss]: There is no Logo component, I will make one then

File: src/components/Footer/index.scss [hunk 6]
```diff
@@ -61,22 +63,21 @@ $Logo-padding: 2px;
...
 .Footer-logo {
-  padding: $Logo-padding;
+  padding: 2px;
 
   background-color: $color-yellow;
   color: $color-deep-blue;
 
   font-family: $font-family-mono;
-  font-size: $Logo-font-size-small;
+  font-size: 12px;
```
Comment (frm) [inline src/components/Footer/index.scss]: `$size-xsmall`. Other than this, seems ready to go

**Review by frm — CHANGES_REQUESTED** [review]

**Review by frm — APPROVED** [review]

**Comment by jfranciscosousa** [comment]
@fribmendes done i think, fixed an issue with the state and the scroll anim. A few tests on different browsers tomorrow and it's done.

File: src/components/Footer/index.scss (hunk 6 above)
Comment (jfranciscosousa) [reply src/components/Footer/index.scss]: 👍

**Comment by frm** [comment]
LGTM. Whenever you feel comfortable, feel free to merge 👍

**Comment by frm** [comment]
PS: I just added you to the Semaphore org. Can you transfer the project there?

**Comment by jfranciscosousa** [comment]
Sure thing

**Pull request merged**